lib/mysql_rewinder/cleaner.rb

```diff
@@ -31,5 +35,22 @@ def all_tables
         WHERE TABLE_SCHEMA = DATABASE()
       SQL
     end
+
+    private
+
+    def log_sql(sql)
+      return yield unless @logger.debug?
```

Comment:
With this, I understand that the logging code below is executed even if `@logger` is NullLogger, that isn't desirable.

Instead, we should set `@logger = nil` if logger is not passed.
Then we can write like:

```ruby
def log_and_execute(sql)
  return @client.execute(sql) unless @logger&.debug?

  ...
end
```

Reply:
Thanks. Fixed for https://github.com/DeNA/mysql_rewinder/pull/4/commits/69070ff65426744b12b767cd63df71029bed3bc5